crates/fluvio-connector-package/test-data/connectors/error-batchsize.yaml

```diff
@@ -1,3 +1,4 @@
+schemaVersion: v1
```

Comment:
We are using `apiVersion` in other configs like connector and SmartModule.   I suggest we keep the same.  Also they are using server which seems to be more easy to manage

Reply:
Got it, In the issue I put apiVersion, but then changed because that name seemed more accurate to me but I don't have any strong opinion about it. Changing back to `apiVersion`

Reply:
`schemaVersion` will be confused with the new schema mgt feature

Reply:
and use semversion